descartes_light/solvers/include/descartes_light/solvers/ompl/ompl_solver.h

```diff
@@ -0,0 +1,72 @@
+#ifndef DESCARTES_LIGHT_SOLVERS_OMPL_OMPL_SOLVER_H
+#define DESCARTES_LIGHT_SOLVERS_OMPL_OMPL_SOLVER_H
+
+#include <descartes_light/solvers/bgl/bgl_solver.h>
+#include <descartes_light/solvers/ompl/descartes_space.h>
+#include <ompl/base/Planner.h>
+#include <ompl/geometric/SimpleSetup.h>
+
+namespace descartes_light
+{
+/**
+ * @brief BGL solver implementation that constructs vertices and edges in the build function and uses Dijkstra's
+ * algorithm with a default visitor to search the graph
+ */
+template <typename FloatType>
+class BGLOMPLSolver : public BGLSolverBaseSVDE<FloatType>
```

Comment:
Per the previous comment, we shouldn't build this off of the BGL solver. It might make sense to inherit from the `LadderGraphSolver` though

Reply:
It now inherits directly from `Solver`